### DISCUSSION TITLE
开发者例会议题收集（2022-11-15）

### DISCUSSION THREAD

**Discussion by chuntaojun** [polarismesh/polaris#808] [Meeting]
# 开发者例会议题收集（2022-11-15）

## 会议信息

andrewshan(单家骏) 邀请您参加腾讯会议
会议主题：PolarisMesh社区双周例会
会议时间：2022/11/15 20:00-20:30 (GMT+08:00) 中国标准时间 - 北京
重复周期：2022/11/15-2023/03/21 20:00-20:30, 每两周 (周二)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/dm/bi26r1VIAx2y

#腾讯会议：825-9154-7185

## 会议录屏信息

主题: PolarisMesh社区双周例会
日期: 2022-11-15 19:56:27
录制文件：https://meeting.tencent.com/v2/cloud-record/share?id=c5662d42-bbe0-4e50-aa4b-a976f94c3891&from=3

**Comment by edocevol** [comment]
@edocevol 分享 北极星旁路功能插件设计——支持 Local Plugin 以及 Remote Plugin 模式 #757 实现及使用 Demo

**Reply by chuntaojun** [reply]
> @edocevol 分享 北极星旁路功能插件设计——支持 Local Plugin 以及 Remote Plugin 模式 #757 实现及使用 Demo

新增任务

前端需要考虑如何支持页面模块化，即同一个页面的不同实现，做成不同的依赖，然后在构建期间根据相关配置文件决定不同的前端页面模块依赖下载并进行构建

**Reply by chuntaojun** [reply]
@andrewshan 与社区同学一同统一 Local 以及 伴生模式下的插件设计

**Comment by andrewshan** [comment]
@andrewshan 分享 北极星java-agent框架的设计，https://github.com/polarismesh/polaris-java-agent/issues/88

**Comment by shichaoyuan** [comment]
@shichaoyuan  讨论一下 polaris-server leader 选举功能模块， https://github.com/polarismesh/polaris/issues/789

**Reply by gtyi999** [reply]
@andrewshan 分享下 接下来的北极星roadmap, 路线和规划。

**Reply by chuntaojun** [reply]
> @andrewshan 分享下 接下来的北极星roadmap, 路线和规划。

这个可以先提一下您这里期望北极星还具备哪些能力
